File: CS/FileList/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.IO;
using DevExpress.XtraTreeList.Nodes;
using DevExpress.XtraTreeList;
using System.Diagnostics;

namespace FileList
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            new FileListHelper(treeList1);
        }

    }
}

File: VB/FileList/Form1.vb
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports System.Diagnostics

Namespace FileList

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            Dim tmp_FileListHelper = New FileListHelper(treeList1)
        End Sub
    End Class
End Namespace
